client/src/components/CheckboxSelector.js

```diff
@@ -0,0 +1,55 @@
+import './CheckboxSelector.scss';
+
+export class CheckboxSelector extends React.Component {
+  constructor(props){
+    super(props);
+  }
+  componentDidUpdate(){
+  }
+  render(){
+    const { items, handleToggle } = this.props;
+    return (
+      _.map(items, (item, key) =>
+        <Checkbox key={key} toggleHandler={handleToggle} {...item}/>
+      )
+    );
+  }
+}
+
+export class Checkbox extends React.Component {
+  constructor(props){
+    super(props);
+    this.state = {
+      checked: props.active,
+    };
+  }
+  render(){
+    const {id, label, color} = this.props;
+    const checked = this.state.checked;
+    const onClickHandler = () => {
+      const newChecked = !this.state.checked;
+      this.props.toggleHandler(this.props.id,newChecked);
+      this.setState({checked: newChecked});
+    };
+    const borderStyle = {borderColor: color};
+    return (
+      <div className="checkbox-selector">
+        <button
+          className={"checkbox-selector--box"}
+          style={checked ? { backgroundColor: color, ...borderStyle} : borderStyle }
+          tabIndex={0}
+          aria-pressed={checked}
+          aria-labelledby={"checkboxLabel_"+id}
+          onClick={onClickHandler}
+        />
+        <label
+          className="checkbox-selector--label"
+          id={"checkboxLabel"+id}
+          onClick={onClickHandler}
+        >
+          {label}
+        </label>
+      </div>
```

Comment:
Not playing well with mac voiceover, labels and checked/unchecked state not being read. 

For labels, it's possible voiceover doesn't support `aria-labelledby`. Try giving the button an id and the `label` element a `for={id}` attribute.

For the state not being read, I'm not sure, but maybe my earlier suggestion about the role and `aria-checked` together will fix that, will need to test that though.

Reply:
Oh, wait, on closer inspection, there's an underscore in the `aria-labelledby` id construction that's not in the label id. Should DRY that.

Nitpick, but while you're fixing that, out convention is for ids to be in kebab-case.